rpm/clang.spec

```diff
@@ -3,32 +3,20 @@
 %global patch_ver 7
 
 %global clang_tools_binaries \
-	%{_bindir}/clang-apply-replacements \
-	%{_bindir}/clang-change-namespace \
 	%{_bindir}/clang-check \
-	%{_bindir}/clang-doc \
 	%{_bindir}/clang-extdef-mapping \
 	%{_bindir}/clang-format \
-	%{_bindir}/clang-include-fixer \
 	%{_bindir}/clang-linker-wrapper \
-	%{_bindir}/clang-move \
 	%{_bindir}/clang-nvlink-wrapper \
 	%{_bindir}/clang-offload-bundler \
 	%{_bindir}/clang-offload-packager \
 	%{_bindir}/clang-offload-wrapper \
-	%{_bindir}/clang-pseudo \
-	%{_bindir}/clang-query \
 	%{_bindir}/clang-refactor \
-	%{_bindir}/clang-reorder-fields \
 	%{_bindir}/clang-rename \
 	%{_bindir}/clang-repl \
 	%{_bindir}/clang-scan-deps \
-	%{_bindir}/clang-tidy \
-	%{_bindir}/clangd \
 	%{_bindir}/diagtool \
-	%{_bindir}/hmaptool \
-	%{_bindir}/pp-trace \
-	%{_bindir}/run-clang-tidy
+	%{_bindir}/hmaptool
```

Comment:
Why did you remove those? Are you sure those are not needed or useful?

Reply:
FWIW, rpm would throw an error if those actually did exist and were not listed.

Reply:
I suspect this is because you forgot to run %prep section of the section which adds symlink to enable build of extra tools.

Reply:
The broken commit is removed.